scripts/lint/metadata.ts

```diff
@@ -8,6 +8,7 @@ import { relative } from "std/path/mod.ts";
 import { REPO_ROOT } from "@/deps.ts";
 import { log } from "@/lint/logger.ts";
 import { formatListOfItems, getUserstylesData } from "@/utils.ts";
+import stringify from "npm:json-oneline-stringify";
```

Comment:
```suggestion

const stringify = (v: unknown) => JSON.stringify(v, null, 0);
```

Reply:
I mean it's less readable... for example it doesn't space apart arrays or objects.
![image](https://github.com/catppuccin/userstyles/assets/47499684/2a08e4cb-530f-458e-bf12-df136bf15abe)

Reply:
don't think we should care for this lint output

Reply:
I think we should because making these warnings readable is pretty key.

![Screenshot 2024-02-17 at 06 29 03 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/8b5f8b7e-51e6-47c8-b4f9-b37ec1b4ed02)